src/doc/book/conditional-compilation.md

```diff
@@ -41,8 +41,12 @@ they get set in the [`[features]` section][features] of your `Cargo.toml`:
 # no features by default
 default = []
 
+# Add feature "foo" here, then you can use it. 
+# Our "foo" feature depends on nothings else.
+foo = []
+
 # The “secure-password” feature depends on the bcrypt package.
-secure-password = ["bcrypt"]
+# secure-password = ["bcrypt"]
```

Comment:
Ah so, as the comment says

> depends on the bcrypt package

If you don't have the package as a dependency, then it's going to give an error. Can we just remove these three lines?